static/js/util/util.js

```diff
@@ -50,6 +51,38 @@ export function makeCourseStatusDisplay(course, now = moment()) {
     courseOrRun = course.runs[0];
   }
 
+  let onEnrollBtnClick = (event) => {
+    event.preventDefault();
+    let url = [];
+    if (courseOrRun.course_id) {
+      url.push(
+        SETTINGS.edx_base_url,
+        'courses/',
+        courseOrRun.course_id,
+        '/info'
+      );
+      window.open(url.join(""), '_blank');
+    } else {
+      alert("Please enter valid course id");
```

Comment:
- is there always course id?
- if not the what kind of validation message should we show?